### ISSUE TITLE
[BUG]: possible bug with multiclass MCC code?

### ISSUE THREAD

**Issue by davidegraff** [chemprop/chemprop#331]
**Describe the bug**
I think the multiclass MCC loss is returning the native multiclass MCC value *rather* than the actual MCC loss, which should be `1-MCC`

**Example(s)**
see [here](https://github.com/chemprop/chemprop/blob/master/chemprop/train/loss_functions.py#L111), which is exactly the same as the `sklearn` implementation. The native MCC value is not the correct loss value because we want to _maximize_ MCC but the optimization routine attempts to _minimize_ the loss value, meaning that it is working opposite as intended.

**Expected behavior**
That we _maximize_ the MCC value. I think it should be returning `1 - [...]`

**Comment by cjmcgill** [comment]
@davidegraff yes that is exactly what has happened. And this explains the behavior/failure I saw using multiclass MCC just last week. Will make a pull.

**Comment by cjmcgill** [comment]
@kevingreenman all good. Thanks for fixing my original implementation of it. Haha I was wondering why it felt different than my other code.
